packages/fields/types/Content/blocks/blockquote.js

```diff
@@ -0,0 +1,6 @@
+const path = require('path');
```

Comment:
I started out with all this in the `index.js`, but broke them out into individual files like this once I realised I need the `paragraph.js` for setting the default in `Implementation.js`.

I don't mind them being split out into separate files like this; it's reminiscent of the field types themselves. OTOH; I'm not against moving them back into `index.js`, it just means we'll end up with duplicate definitions for `paragraph.js` which is a bit icky.